code/game/machinery/body_scanner.dm

```diff
@@ -565,7 +565,8 @@
 				else if(istype(I, /obj/effect/spider))
 					organic += I
 				else
-					unk += 1
+					if(!istype(I, /obj/item/implant/uplink))
+						unk += 1
```

Comment:
I think we're at a point where it's convenient to just put a variable on implants that determines if they're hidden or not, rather than the chain of istypes that this design pattern would create

Reply:
true but I guess this works for now